pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -203,6 +203,7 @@ public ServerType getServerType() {
 
   public static class Server {
     public static final String CONFIG_OF_INSTANCE_DATA_DIR = "pinot.server.instance.dataDir";
+    public static final String CONFIG_OF_CONSUMER_DIR = "pinot.server.instance.consumerDir";
```

Comment:
suggest "pinot.server.instance.realtime.consumerDir" similar to CONFIG_OF_REALTIME_OFFHEAP_ALLOCATION